server/app/services/applicant/question/NumberQuestion.java

```diff
@@ -33,6 +33,17 @@ public ImmutableList<Path> getAllPaths() {
     return ImmutableList.of(getNumberPath());
   }
 
+  @Override
+  public ImmutableMap<Path, Long> getJsonEntries() {
+    Path path = getNumberPath().asApplicationPath();
+
+    if (getNumberValue().isPresent()) {
+      return ImmutableMap.of(path, getNumberValue().get());
+    } else {
+      return ImmutableMap.of();
+    }
+  }
+
```

Comment:
Above code is taken from `case NUMER`:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/export/JsonExporter.java#L121-L219